src/lib/Constants.ts

```diff
@@ -64,6 +64,7 @@ export const DataType = {
 
 /** CSS class and id constants. */
 export const CSS = {
+  // TODO(cjcox): Refactor out these MDL classes in favor of flexbox.
   MDL_LIST:               "mdl-list",
```

Comment:
Thanks. I filed an [issue](https://github.com/material-foundation/material-remixer-js/issues/89) where I'll handle this for entire app.